search-app/src/cmr/search/middleware/shapefile_simplification.clj

```diff
@@ -263,15 +241,15 @@
   to point to the new (reduced) shapefile"
   [request]
   (when (= "true" (get-in request [:params "simplify-shapefile"]))
-    (if-let [tmp-file (get-in request [:params "shapefile" :tempfile])]
+    (if (get-in request [:params "shapefile" :tempfile])
       (simplify (get-in request [:params "shapefile"]))
       (errors/throw-service-error :bad-request "Missing shapefile"))))
 
 (defn shapefile-simplifier
   "Adds shapefile simplification header to response when shapefile simplication was
   requested."
   [handler default-format-fn]
-  (fn [{context :request-context :as request}]
+  (fn [{_context :request-context :as request}]
```

Comment:
Is it necessary to keep this if it's not used?

Reply:
Yes, this is middleware that must conform to a standard function signature of `[context request]`.